apps/src/code-studio/pd/foorm/editor/foormEditorRedux.js

```diff
@@ -82,10 +87,14 @@ export default function foormEditorRedux(state = initialState, action) {
     return {
       ...state,
       formQuestions: action.formData['questions'],
+      questions: action.formData['questions'],
```

Comment:
Maybe ignore the duplicate Redux store updates here -- I have a follow-up PR (branched off of this one) that has the final state of the Redux changes I'm proposing to make this more generic (wanted to make this PR more digestible, so decided to put Redux changes separately).